skins/standard/global.css

```diff
@@ -1566,10 +1566,6 @@ body {
     font-size: small;
 }
 
-pre {
```

Comment:
This seems like it could have much wider effects and isn't obviously linked to the font size change. Have we vetted all `<pre>` uses to ensure that new alignment issues are not introduced by this change?